### DISCUSSION TITLE
Inquiry Regarding Recent Developments and Future Plans

### DISCUSSION THREAD

**Discussion by bugxzhu** [spacemeshos/go-spacemesh#5857] [General]
Dear Spacemesh Team,

I hope this message finds you well. First and foremost, I want to express my gratitude for the team's contributions to the advancement of decentralized blockchain technology. Your dedication to the principles of decentralization and innovation is truly commendable.

Recently, I've come across some speculations within the community, and I'd like to seek clarification from the official team regarding a few matters.

Regarding the listing of the SMH token on exchanges, I'm curious to know if the team has plans to consider this after the unlocking date. Will there be any official stance or considerations on this matter?

Is the team currently focused on the release of the Virtual Machine (VM)? Additionally, are there any plans to introduce a staking mechanism?

Is Spacemesh considering the launch of storage projects on the Layer 2 (L2) level, similar to projects like Filecoin and Arweave?

I appreciate any insights or updates you can provide on these topics. Your efforts in building and nurturing the Spacemesh ecosystem are highly valued, and I look forward to your response.

Thank you for your time and attention to these inquiries.

Best regards

**Comment by lrettig** [comment]
Thanks for the questions.

> Is the team currently focused on the release of the Virtual Machine (VM)?

Yes, the VM and programmability is one major area of focus. We announced a bunch more about our plans recently: https://spacemesh.io/blog/introducing-athena/.

> Additionally, are there any plans to introduce a staking mechanism?

Not at this time. The Spacemesh consensus and sybil resistance mechanisms are based on hard drive space, not staked assets. Of course staking could be introduced as part of a layer built on top of Spacemesh in the future, but we have no plans to build something like this.

> Is Spacemesh considering the launch of storage projects on the Layer 2 (L2) level, similar to projects like Filecoin and Arweave?

"We" as a team are not but others are free to build these things on Athena (the VM)!

Feel free to post other questions here.

L

**Reply by bugxzhu** [reply]
thanks
